src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -47,15 +47,15 @@ public EditCommand parse(String args) throws ParseException {
             editStudentDescriptor.setName(ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get()));
```

Comment:
Is the .get() needed here? (since it has been removed from the next few lines)

Reply:
Yes, ParseName takes in Name instead of Optional<Name> unlike the rest of the classes (Optional<Classes>). This is due to the fact that the Student's name is always taken in when using the AddCommandParser.